public_html/wp-content/themes/pattern-directory/functions.php

```diff
@@ -80,7 +106,13 @@ function enqueue_assets() {
 
 		wp_add_inline_script(
 			'wporg-pattern-script',
-			sprintf( "var wporgLocale = '%s';", wp_json_encode( get_locale() ) ),
+			sprintf(
+				"var wporgLocale = JSON.parse( decodeURIComponent( '%s' ) )",
+				rawurlencode( wp_json_encode( array(
+					'baseId' => get_locale(),
```

Comment:
I think JavaScript API's call this the `baseName`. I'm not bothered by `baseId` though.

Reply:
I do like `baseName` more, but after looking into this [doc](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale), I found that in its Parameters and Examples sections, it suggests that `ko-Kore-KR` is different from `ko-KR`. 
The former one is `baseName` and the latter one is `locale identifier` (see the description of tag under [Parameters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/Locale#parameters) and this [Example](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/Locale#basic_usage)). 
So I'll change it to just `id` then. Thanks for bringing this up!

Reply:
> So I'll change it to just id then. Thanks for bringing this up!

Done chaning.